dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -186,39 +196,38 @@ private LinkageChecker(
 
     // Filter classes in whitelist
     SetMultimap<SymbolProblem, ClassFile> filteredMap =
-        Multimaps.filterEntries(problemToClass.build(), LinkageChecker::problemFilter);
+        Multimaps.filterEntries(problemToClass.build(), this::problemFilter);
     return ImmutableSetMultimap.copyOf(filteredMap);
   }
 
+  private static ImmutableList<LinkageErrorMatcher> readExclusionMatchers() throws IOException {
+    ImmutableList.Builder<LinkageErrorMatcher> exclusionMatchers = ImmutableList.builder();
+
+    try {
+      URL defaultRuleUrl = ClassLoader.getSystemResource("linkage-checker-exclusion-default.xml");
+      ImmutableList<LinkageErrorMatcher> defaultMatchers =
+          ExclusionFileParser.parse(defaultRuleUrl);
+      exclusionMatchers.addAll(defaultMatchers);
+    } catch (SAXException | VerifierConfigurationException ex) {
+      throw new IOException("Could not read default exclusion rule", ex);
+    }
+
+    return exclusionMatchers.build();
+  }
+
   /**
    * Returns true if the linkage error {@code entry} should be reported. False if it should be
    * suppressed.
    */
-  private static boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry) {
-    ClassFile classFile = entry.getValue();
+  private boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry) {
```

Comment:
This method is non-static because exclusionMatchers field is an instance variable.